var/spack/repos/builtin/packages/openexr/package.py

```diff
@@ -32,9 +32,8 @@ class Openexr(AutotoolsPackage):
     version('1.3.2', sha256='fa08ad904bf89e2968078d25d1d9817f5bc17f372d1bafabf82e8f08ca2adc20',
             url="http://download.savannah.nongnu.org/releases/openexr/openexr-1.3.2.tar.gz")
 
-    variant('debug', default=False,
-            description='Builds a debug version of the libraries')
-
+    variant('debug', default=False, description='Builds a debug version of the libraries')
+    depends_on('cmake@3.12.0:', type='build')
```

Comment:
I assume you're adding this for the version constraint
I suggest adding a comment pointing to https://github.com/AcademySoftwareFoundation/openexr/blob/master/INSTALL.md#prerequisites